### PR TITLE
New article Create-control-binding-4

### PR DESCRIPTION
**Pull request by rachana-satao**
US#1888235

This article is merging of - [link](https://docs.microsoft.com/en-us/dotnet/desktop/winforms/how-to-create-a-simple-bound-control-on-a-windows-form?view=netframeworkdesktop-4.8), [link](https://docs.microsoft.com/en-us/dotnet/desktop/winforms/how-to-create-a-bound-control-and-format-the-displayed-data?view=netframeworkdesktop-4.8)

Preview-  [link](https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/data/how-to-create-control-binding?view=netdesktop-5.0&branch=pr-en-us-1392)

### CODE REVIEW

**Comment by rachana-satao** [comment]
@adegeo Acrolinx score for this article - 96

File: dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+01. Select the name of an element to bind to.
+
+01. If you're working in the **Formatting and Advanced Binding** dialog, select **OK** to return to the **Properties** window.
+
+01. If you want to bind more properties of the control, repeat steps 3 through 7.
+
+    > [!NOTE]
+    > As simple-bound controls show only a single data element, it's very typical to include navigation logic in a Windows Form with simple-bound controls.
+
+## How to Create a Bound Control and Format the Displayed Data
```
Comment (Lovina-Saldanha) [inline dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md:42]: The heading format is different from the rest. Others have sentence case.

**Comment by rachana-satao** [comment]
@adegeo Acrolinx score for this article - 95

File: dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+01. Expand the data source you want to bind to until you find the single data element you want.
+
+     For example, if you're binding to a column value in a dataset's table, expand the name of the dataset, and then expand the table name to display column names.
+
+01. Select the name of an element to bind to.
+
+01. In the **Format type** box, select the format you want to apply to the data displayed in the control.
+
+     In every case, you can specify the value displayed in the control if the data source contains <xref:System.DBNull>. Otherwise, the options vary slightly, depending on the format type you select. The following table shows the format types and options.
```
[the remark this answers is not in the document]
Comment (rachana-satao) [reply dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md:64]: It is changing the paragraph style. Maintaining text separation here.

File: dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## How to simple bind a control
+
+01. Connect to a data source. For more information, see [Connecting to a Data Source](/dotnet/framework/data/adonet/connecting-to-a-data-source).
+
+01. In Visual Studio, select the control on the form and display the **Properties** window.
+
+01. Expand the **(DataBindings)** property.
+
+     The properties that bound are displayed underneath the **(DataBindings)** property. For example, in most controls, the **Text** property is frequently bound.
```
[the remark this answers is not in the document]
Comment (rachana-satao) [reply dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md:23]: Removing black line will change preview of the article.

File: dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+01. If the property you want to bind isn't one of the commonly bound properties, select the **Ellipsis** button (:::image type="content" source="./media/how-to-create-control-binding/visual-studio-ellipsis-button.png" alt-text="The Ellipsis button in the Properties window of Visual Studio.":::) in the **(Advanced)** box to display the **Formatting and Advanced Binding** dialog with a complete list of properties for that control.
+
+01. Select the property you want to bind and select the drop-down arrow under **Binding**.
```
[the remark this answers is not in the document]
Comment (rachana-satao) [reply dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md:27]: when you click on binding property, the list of available data sources is displayed.

File: dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+01. Connect to a data source. For more information, see [Connecting to a Data Source](/dotnet/framework/data/adonet/connecting-to-a-data-source).
+
+01. In Visual Studio, select the control on the form, and then open the **Properties** window.
+
+01. Expand the **(DataBindings)** property, and then in the **(Advanced)** box, select the ellipsis button (:::image type="content" source="./media/how-to-create-control-binding/visual-studio-ellipsis-button.png" alt-text="The Ellipsis button in the Properties window of Visual Studio.":::) to display the **Formatting and Advanced Binding** dialog, which has a complete list of properties for that control.
+
+01. Select the property you want to bind, and then select the **Binding** arrow.
+
+     A list of available data sources is displayed.
```
[the remark this answers is not in the document]
Comment (rachana-satao) [reply dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md:54]: It is a result of the step. Separation is needed.

File: dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (adegeo) [reply dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md:61]: I agree with @rachana-satao

File: dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+01. [Connect to a data source](/dotnet/framework/data/adonet/connecting-to-a-data-source).
+
+01. In Visual Studio, select the control on the form and then open the **Properties** window.
+
+01. Expand the **DataBindings** property, and then in the **Advanced** box, select the ellipsis button (:::image type="content" source="./media/how-to-create-control-binding/visual-studio-ellipsis-button.png" alt-text="The Ellipsis button in the Properties window of Visual Studio.":::) to display the **Formatting and Advanced Binding** dialog, which has a complete list of properties for that control.
+
+01. Select the property you want to bind, and then select the **Binding** arrow.
+
+     A list of available data sources is displayed.
```
Comment (adegeo) [inline dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md:51]: Please double check all child content off the numbered items. Some are indented 5 spaces but they should be indented 4

Comment (rachana-satao) [reply dotnet-desktop-guide/net/winforms/data/how-to-create-control-binding.md:51]: Ok

**Comment by adegeo** [comment]
@rachana-satao FYI I have some updated Visual Studio icons in a shared folder to be used by other articles: `dotnet-desktop-guide/net/winforms/media/shared/`

**Review by adegeo — APPROVED** [review]

**Pull request merged**